docs/users_guide/landice/test_groups/ensemble_generator.rst

```diff
@@ -177,29 +243,31 @@ The model configuration uses:
 * ISMIP6 surface mass balance and sub-ice-shelf melting using climatological
   mean forcing
 
-Steps for setting up and running a Thwaites ensmble
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The initial condition and forcing files are specified in the
+``ensemble_generator.cfg`` file or a user modification of it.
+
+Steps for setting up and running an ensmble
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 1. With a compass conda environment set up, run, e.g.,
```

Comment:
add -p flag for exe. Note that -f is not required if you use the default cfg file.

Reply:
I left off the `-p` flag because I meant this as an example, and I didn't want to explain all the variations in how `compass setup` could be invoked.  I would actually recommend the MALI submodule be used for ensembles (and therefore the `-p` option not be used) to have a dedicated executable with the ensemble generation framework (which has been a moving target so far).  On the other hand, I explicitly included the `-f` option here because this step of the instructions recommends users create a custom cfg file and not use the default unmodified.

Reply:
okay, that's fine with me.